src/index.ts

```diff
@@ -1,42 +1,56 @@
 import Client from './client';
+import LRU from 'lru-cache';
 import { Options, Target } from './types';
 import { Logger } from './log';
-import { KeyValueStore } from './types';
+import { AsyncKeyValueStore, KeyValueStore } from './types';
+import { FileStore } from './store';
 
-export { Client, Options, Target, KeyValueStore, Logger };
+export {
+  Client,
+  Options,
+  Target,
+  AsyncKeyValueStore,
+  KeyValueStore,
+  Logger,
+  LRU,
+  FileStore,
+};
 export default {
   instance: undefined,
   init: function (sdkKey: string, options: Options): void {
     if (!this.instance) {
       this.instance = new Client(sdkKey, options);
     }
   },
-  boolVariation: function (
+  boolVariation: async function (
     identifier: string,
     target: Target,
     defaultValue = false,
-  ): boolean {
+  ): Promise<boolean> {
     return this.instance.boolVariation(identifier, target, defaultValue);
```

Comment:
Does `this` work in this context? i.e. not inside a class? Might be worth checking it out. The `this.instance = new Client` on line 22 might be actually pointing to the global scope, not this exported object, so these `this.instance.*` calls might only be working because they too are referencing the global scope.

If you change one of these functions to use the method syntax, you may be able to prove that one way or the other.

```typescript
export default {
  // ...
  
  async boolVariation(
    identifier: string,
    target: Target,
    defaultValue = false,
  ): Promise<boolean> {
    return this.instance.boolVariation(identifier, target, defaultValue);
  },

  // ...
}
```

Reply:
Never mind, it was driving my IDE crazy